.github/workflows/site.yml

```diff
@@ -14,7 +14,7 @@ name: Website
 jobs:
   build:
     name: Build and Test
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Temporarily until zio-sbt is fixed, sbt is missing in 24.04

Reply:
Yet another option to solve that issue but for `ubuntu-latest` is [using of coursier/setup-action@v1 instead of actions/setup-java@v4](https://github.com/plokhotnyuk/jsoniter-scala/commit/7dddd1ae1fcfc3862c3dd525898ed26d7bf2c542#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R41).